src/scales/scale.time.js

```diff
@@ -175,8 +175,23 @@ module.exports = function(Chart) {
 		var ticks = [];
 		if (options.maxTicks) {
 			var stepSize = options.stepSize;
-			ticks.push(options.min !== undefined ? options.min : niceRange.min);
-			var cur = moment(niceRange.min);
+			var startTick = options.min !== undefined ? options.min : niceRange.min;
+			var seniorUnitStart = moment(startTick).add(1, options.seniorUnit).startOf(options.seniorUnit);
```

Comment:
`majorUnit` is auto-detected on line #340 and then passes to `options` on line #353